### ISSUE TITLE
[libuv] all builds fail with "the 'build only if scm changes' feature is enabled."

### ISSUE THREAD

**Issue by saghul** [nodejs/build#457]
~~~~
Started by user Saúl Ibarra Corretgé
[EnvInject] - Loading node environment variables.
Building on master in workspace /var/lib/jenkins/jobs/libuv-test-commit/workspace
    >> Job status: [libuv-test-commit-osx] the 'build only if scm changes' feature is enabled.
Starting build job libuv-test-commit-osx.
    >> Job status: [libuv-test-commit-freebsd] the 'build only if scm changes' feature is enabled.
Starting build job libuv-test-commit-freebsd.
    >> Job status: [libuv-test-commit-linux] the 'build only if scm changes' feature is enabled.
Starting build job libuv-test-commit-linux.
    >> Job status: [libuv-test-commit-smartos] the 'build only if scm changes' feature is enabled.
Starting build job libuv-test-commit-smartos.
    >> Job status: [libuv-test-commit-aix] the 'build only if scm changes' feature is enabled.
Starting build job libuv-test-commit-aix.
    >> Job status: [libuv-test-commit-windows] the 'build only if scm changes' feature is enabled.
Starting build job libuv-test-commit-windows.
Finished Build : #71 of Job : libuv-test-commit-osx with status : FAILURE
Finished Build : #49 of Job : libuv-test-commit-freebsd with status : FAILURE
Finished Build : #78 of Job : libuv-test-commit-linux with status : FAILURE
Finished Build : #75 of Job : libuv-test-commit-smartos with status : FAILURE
Finished Build : #85 of Job : libuv-test-commit-aix with status : FAILURE
Finished Build : #91 of Job : libuv-test-commit-windows with status : FAILURE
Notifying upstream projects of job completion
Warning: you have no plugins providing access control for builds, so falling back to legacy behavior of permitting any downstream builds to be triggered
Finished: FAILURE
~~~~

**Comment by joaocgreis** [comment]
Hi! This is unrelated to "build only if scm changes" message. There's a space in the target repository of that job, so git tries to fetch `https://github.com/ h0tw1r3/libuv.git/` and fails. Here is another run with the same parameters: https://ci.nodejs.org/view/libuv/job/libuv-test-commit/78/

About the "build only if scm changes", that is an issue with Jenkins. We had to enable it because a plugin update inverted the logic. So it says that the feature is enabled but it is actually disabled. If we disabled it, it would only build when it detected changes.
